namer/zk-leader/src/main/java/com/twitter/common/zookeeper/ZooKeeperClient.java

```diff
@@ -0,0 +1,518 @@
+// =================================================================================================
```

Comment:
can you use https://github.com/twitter/finagle/blob/develop/finagle-serversets/src/main/java/com/twitter/finagle/common/zookeeper/ZooKeeperClient.java?